oaknational/Oak-Web-Application
# Added styling to curriculum explainers in docx

## Description
Added styling to curriculum explainers in docx, also

 - Disabled list of thread for cycle 2
 - Removed subject principals for cycle 2

## Issue(s)

Fixes `CUR-894`

## How to test
Cycle 1

1. Go to https://deploy-preview-2816--oak-web-application.netlify.thenational.academy
2. Check for regressions in curriculum download (docx)

Cycle 2: enable cycle 2 locally 

1. Go to https://deploy-preview-2816--oak-web-application.netlify.thenational.academy
2. Check explainers curriculum download (docx) display correctly
